src/components/task-container/TaskContainer.tsx

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 import {connect} from 'react-redux';
 import Task from '../task/Task';
 import {fetchTaskSuccess} from '../redux/Actions/TaskAction';
+import {Gallery} from '@patternfly/react-core';
+import './index.css';
 
 export interface MockData{
   Name : string,
   Description : string,
   Rating : number,
   Downloads : number,
-  YAML : string
+  YAML : string,
+  TAGS : [],
```

Comment:
Try following the same interface cases everywhere. Tags | TAGS | tags ?
Use smaller case letters